### DISCUSSION TITLE
Non-rigidbody Collision Detection

### DISCUSSION THREAD

**Discussion by accmltr** [dyn4j/dyn4j#289] [Q&A] [answered]
Hi, 

Is there a way to create shapes to detect other shapes, without being a part of the physics simulation? Something like the Area2D Node in Godot?

Any advice/help would be greatly appreciated.

God bless,
Accumulator

**Comment by wnbittle** [comment, marked as answer]
Yes.  There's a few ways to do this.

The first is to use the `detect` methods of the `World` class.  These methods will accept an `AABB` or `Convex`.

The second way is to create a `Body` and as you add the `Fixture`s, set each of them to a sensor using `setSensor`.  Then you can get collision notifications from the world listeners.

**Reply by accmltr** [reply]
Thank you very much. I am yet to get started on integrating this into my project, but I just wanted to be sure before I go for it.

Looking forward to attempting it!
